src/cards/ha-plant-card.html

```diff
@@ -109,7 +109,10 @@
     }
 
     computeUOM(stateObj, sensor) {
-      return _CONFIG[sensor].uom;
+      if (typeof stateObj.attributes.unit_of_measurement[sensor] === 'undefined') {
```

Comment:
This is going to be renamed. `unit_of_measurement` should never be an object.

Reply:
btw, you can check for existence of a property with `if (sensor in stateObj.attributes.units)`

Reply:
I'm a convert to "in" syntax for attribute checking! Much more readable. Both comments addressed in latest commit.